eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -793,7 +793,7 @@ data ImpTestEnv era = ImpTestEnv
       LedgerState era ->
       Tx era ->
       BaseImpM ()
-  , iteCborRoundTripFailures :: !Bool
+  , iteCborRoundTripFailures :: Bool
```

Comment:
This too?  It's not like the other rule-environment data types, derived from the the state or transaction. At the same time, I can't think of a  reason why it would be better for it to be strict , so I'm not opposed to the change, just wondering if there might have been a reason why it was declared like this to begin with.

Reply:
It's only used in tests, so I don't think it matters too much. I probably made it strict because of muscle memory.